ui_advanced_config.py

```diff
@@ -528,9 +528,13 @@ def init_entry_widgets(self):
         # COMPONENT
```

Comment:
there is a problem updating the id of a component. the component associated with the old id is still in the json and a duplicated with the new id is added in addition to the old object. the combobox does not include this duplicate.

Reply:
will do tomorrow

Reply:
Fixed I think

Reply:
Changes for me now, looks good